en/TripUpdate.md

```diff
@@ -11,6 +11,6 @@ General guidelines for trip cancellations:
 | vehicle | refer to [message VehicleDescriptor](#VehicleDescriptor) |
 |  | If separate `VehiclePosition` and `TripUpdate` feeds are provided, [TripDescriptor](#TripDescriptor) and [VehicleDescriptor](#VehicleDescriptor) ID values pairing should match between the two feeds.<br/>For example, a `VehiclePosition` entity has `vehicle_id:A` and `trip_id:4`, then the corresponding `TripUpdate` entity should also have `vehicle_id:A` and `trip_id:4`. If any `TripUpdate` entity has `trip_id:4` and any `vehicle_id` other than 4, this is an error. |
 | stop_time_update | `stop_time_updates` for a given `trip_id` should be strictly ordered by increasing `stop_sequence` and no `stop_sequence` should be repeated. |
-|  | All TripUpdates should include at least one `stop_time_update` with a predicted arrival or departure time in the future. If all `stop_time_updates` for a trip reference past arrival and departure times, consumers should assume that no real-time data is available for the trip. |
+|  | All TripUpdates should include at least one `stop_time_update` with a predicted arrival or departure time in the future or recent past. If all `stop_time_updates` for a trip reference past arrival and departure times, consumers should assume that no real-time data is available for the trip. `stop_time_updates` in the recent past can be used to notify consumers that a vehicle has served the trip earlier than expected. |
```

Comment:
I 100% agree with the last sentence (and the spec actually requires it of producers).

For the first sentence, could we change it to:

>All TripUpdates should include at least one `stop_time_update` with a predicted arrival or departure time in the future while the trip is in progress

This sentence was intended to address scenarios I've seen where the stop_time_updates trail behind the current time and presumably are actually observed schedule deviation values.

So if it's 10:00am, you'd see a header timestamp of 9:55am and stop_time_update values like:
* stop 1 - 9:50am
* stop 2 - 9:55am

and then at 10:05am you'd see a header timestamp of 10am and:
* stop 2 - 9:55am
* stop 3 - 10am

Related - I'd like to see TU.stop_time_updates be clarified to be only predictions, and TU.delay to be deprecated and replaced with VP.schedule_deviation, which would be for observed deviation from the schedule (not predictions).

Reply:
> All TripUpdates should include at least one `stop_time_update` with a predicted arrival or departure time in the future while the trip is in progress

@barbeau Even if the trip isn't currently in progress (ended early) we'd still want to keep it around if it was scheduled to be running. Maybe a parenthetical?

> All TripUpdates should include at least one `stop_time_update` with a predicted arrival or departure time in the future while the trip is in progress (or scheduled to be in progress).

Reply:
@paulswartz Yes, my proposed text definitely wasn't intended to exclude that case - your new proposed text works for me!